packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -264,7 +262,12 @@ export async function load_node({
 								fetched.push({
 									url: requested,
 									body: /** @type {string} */ (opts.body),
-									json: `{"status":${status_number},"statusText":${s(response.statusText)},"headers":${s(headers)},"body":${escape_json_in_html(body)}}`
+									json: {
```

Comment:
love that this asymetric escape json string build is being refactored!

now it's no longer a json string, is there a better name for it? 

Unfortunately i think there was a reason behind the different treatment of headers and body. It may no longer be relevant due to the changes in escaping but in a previous version applying the same escaping rules to headers meant that there were cases where header values got mangled that shouldn't have been.

Reply:
This might be worth looking into, then, because I don't see how it'd have altered the output json—after `JSON.parse`, you'd get the same string back, even on the previous implementation.

Maybe the issue was with double-escaping at some point?

Reply:
I git-blamed this down to inception and those were indeed never escaped—I'm just unsure why it would've caused a problem. My search-fu failed me in trying to find a discussion on this.

I believe they are being handled correctly, for what it's worth, but let me know if you see the issue!

Reply:
this comment is the only ref i could find of it https://github.com/sveltejs/kit/pull/2534#issue-1013386805 . basically my initial approach at improving this was encoding the whole `json` object and that failed a testcase. If tests are working now and headers isn't used anywhere else it should be fine.

Are headers even needed on the clientside in the script body? it's not like we can magically turn them back into http headers somehow and what if there was an http-only cookie in there?

Reply:
They are sent for the sake of being included in the `new Response()` we create that can be consumed by `load()` client-side.

But, yeah, tests pass now and I believe this is working.